ui/raidboss/data/06-ew/raid/p10n.ts

```diff
@@ -1,14 +1,201 @@
+      alertText: (data, _matches, output) => {
+        data.silkspitOnYou = true;
+        return output.text!();
+      },
+      outputStrings: {
+        text: {
+          en: 'Away from party and posts',
+        },
+      },
+    },
+    {
+      id: 'P10N Silkspit Others',
+      type: 'HeadMarker',
+      netRegex: { id: '01CE', capture: false },
+      delaySeconds: 0.2,
+      suppressSeconds: 1,
+      infoText: (data, _matches, output) => {
+        if (data.silkspitOnYou === false)
+          return output.text!();
+      },
+      outputStrings: {
+        text: {
+          en: 'Away from marked players',
+        },
+      },
+    },
+    {
+      id: 'P10N Pandaemonic Pillars',
+      type: 'StartsUsing',
+      netRegex: { id: '825D', source: bossNameUnicode, capture: false },
+      durationSeconds: 5,
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Soak towers',
+        },
+      },
+    },
+    {
+      id: 'P10N Imprisonment',
+      type: 'StartsUsing',
+      netRegex: { id: '8262', source: bossNameUnicode, capture: false },
+      suppressSeconds: 1,
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Avoid jails',
+        },
+      },
+    },
+    {
+      id: 'P10N Cannonspawn',
+      type: 'StartsUsing',
+      netRegex: { id: '8264', source: bossNameUnicode, capture: false },
+      suppressSeconds: 1,
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Under jails',
+        },
+      },
+    },
+    {
+      id: 'P10N Ultima',
+      type: 'StartsUsing',
+      netRegex: { id: '827B', source: bossNameUnicode, capture: false },
+      response: Responses.aoe(),
+    },
+    {
+      id: 'P10N Pandaemoniac Meltdown',
+      type: 'StartsUsing',
+      netRegex: { id: '87A2', source: bossNameUnicode },
+      response: Responses.stackMarkerOn(),
+    },
+    {
+      id: 'P10N Soul Grasp',
+      type: 'StartsUsing',
+      netRegex: { id: '8278', source: bossNameUnicode },
+      response: Responses.sharedTankBuster(),
+    },
+    {
+      id: 'P10N Pandaemoniac Ray',
+      type: 'StartsUsing',
+      netRegex: { id: '826[57]', source: bossNameUnicode },
+      infoText: (_data, matches, output) => {
+        // Half-room cleave west (8265) or east (8267)
+        const safeOutput = matches.id === '8265' ? 'east' : 'west';
+        return output[safeOutput]!();
+      },
+      outputStrings: {
+        east: Outputs.getRightAndEast,
+        west: Outputs.getLeftAndWest,
+      },
+    },
+    {
+      id: 'P10N Touchdown',
+      type: 'StartsUsing',
+      netRegex: { id: '8268', source: bossNameUnicode, capture: false },
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Cross bridge / off center platform',
```

Comment:
```suggestion
          en: 'Side Platforms',
```

Or maybe "Get to Side Platform"? Just trying to keep this concise.